packages/contentful--create-contentful-app/README.md

```diff
@@ -5,7 +5,7 @@ This project makes it easy to bootstrap [Contentful Apps](https://www.contentful
 ## Requirements
 
 - Node.js v14 or v16
-- NPM v6
+- npm v6
```

Comment:
I think this was meant to be changed in the --help too. We also need to review all the links there, I fixed one yesterday, not sure about the rest

Reply:
yep, I changed it the help command too. Which links do you mean?

Reply:
https://github.com/contentful/create-contentful-app/blob/f338b5a609b8bdba1d9095a399abc8974025a723/packages/contentful--create-contentful-app/src/index.ts#L132

This is still uppercase ☝️  Also, `Yarn -> yarn` maybe

Links are already updated, you are right :)